src/nickelmenu.cc

```diff
@@ -255,8 +283,19 @@ extern "C" __attribute__((visibility("default"))) void _nm_menu_hook2(MainNavVie
     }
 
     MainNavButton_MainNavButton(btn, parent);
-    MainNavButton_setPixmap(btn, QStringLiteral(":/images/home/main_nav_more.png"));
-    MainNavButton_setActivePixmap(btn, QStringLiteral(":/images/home/main_nav_more_active.png"));
+    MainNavButton_setPixmap(btn, nm_menu_pixmap(
+        nm_global_config_experimental("menu_main_15505_icon"),
+        "/tmp/nm_menu.png",
```

Comment:
Maybe make this a variable to avoid potential typo whoopsies when removing below?

Since you're removing the file, why not reuse the same output filename for both 'normal' and 'active' icons?